truss/templates/shared/logging.py

```diff
@@ -1,15 +1,71 @@
+import json
 import logging
 import sys
+from datetime import datetime
 
+import loguru
 from pythonjsonlogger import jsonlogger
 
 LEVEL: int = logging.INFO
 
+
+def serialize(record):
+    dt = datetime.fromtimestamp(record["time"].timestamp())
+    formatted_time = dt.strftime("%Y-%m-%d %H:%M:%S,%f")[:-3]
+
+    if record["message"][0] == "{":
+        return record["message"]
+
+    subset = {
+        "asctime": formatted_time,
+        "message": record["message"],
+        "levelname": record["level"].name,
+        "request_id": str(record["extra"]["request_id"])
+        if "request_id" in record["extra"]
+        else None,
+    }
+
+    return json.dumps(subset)
+
+
+def patching(record):
+    # we should be able to label different types of logs here
+    record["extra"]["serialized"] = serialize(record)
+
+
+loguru_logger = loguru.logger
```

Comment:
could you leave a comment here explaining what the purpose of these steps are here? what is `loguru_logger.remove()`?